apps/frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./auth-layout.module.css";
```

Comment:
And the same for other styles that you add
```suggestion
import styles from "./styles.module.css";
```